src/Admin/SupportMenu.php

```diff
@@ -0,0 +1,250 @@
+<?php
+/**
+ * Class SupportMenu
+ *
+ * @package Ampproject\Ampwp
+ */
+
+namespace AmpProject\AmpWP\Admin;
+
+use AmpProject\AmpWP\Infrastructure\Conditional;
+use AmpProject\AmpWP\Infrastructure\Registerable;
+use AmpProject\AmpWP\Infrastructure\Service;
+use AmpProject\AmpWP\Services;
+
+/**
+ * SupportMenu class.
+ *
+ * @internal
+ */
+class SupportMenu implements Conditional, Service, Registerable {
```

Comment:
I know we currently have `OptionsMenu` as precedence, but let's improve this by calling it `SupportPage` or better `SupportScreen`.